lib/reporter/tap.js

```diff
@@ -12,7 +12,11 @@ function generateTest(mod, count) {
     mod.error = mod.error.message;
   }
   var error = mod.error ? [directive, mod.error].join(' ') : '';
-  var output = mod.testOutput ? '\n' + util.sanitizeOutput(mod.testOutput, '# ') : '';
+  if (mod.testOutput && mod.testOutput.length > 0) {
+    var output = '\n  ---\n' + util.sanitizeOutput(mod.testOutput, '  #') + '\n  ...';
```

Comment:
does it still need the hash if it is in the yamilish?

Have you tested that this works as expected on more verbose output? I'd very much like to see this run through citgm in CI and see what the output looks like